### PR TITLE
Use cosmos/cosmos-sdk instead of the fork

### PR DESCRIPTION
**Pull request by bluele**

### CODE REVIEW

File: tests/chains/fabric/chaincode/fabibc/go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/datachainlab/ibc-examples/chaincode
+module github.com/hyperledger-labs/yui-relayer/tests/chains/fabric/chaincode/fabibc
```
Comment (siburu) [inline tests/chains/fabric/chaincode/fabibc/go.mod:1]: Is this related to this PR?

Comment (bluele) [reply tests/chains/fabric/chaincode/fabibc/go.mod:1]: @siburu 
Not directly related, but it's a minor fix and I want to include it.

**Review by siburu — APPROVED** [review]
:100:
